### DISCUSSION TITLE
Chat History

### DISCUSSION THREAD

**Discussion by macedo** [tulir/whatsmeow#502] [WhatsApp protocol Q&A]
I'm beginning a new personal project and wondering if is it possible to download/see a chat history. I'm trying to download all media from a group I belong to save to google drive. any thoughts on that?

**Comment by lfaltoni** [comment]
I would also like to know how to do this

**Comment by Heath123** [comment]
> download all media from a group I belong

If that's the only reason you want to do it, WhatsApp does have this built in, you can export chat history including media to a zip file

**Comment by andresovich** [comment]
Anyone found a way to sync on demand the chat history for all conversations?
